src/sinks/websocket/sink.rs

```diff
@@ -301,7 +301,7 @@ impl WebSocketSink {
                         Ok(()) => {
                             finalizers.update_status(EventStatus::Delivered);
 
-                            let message = Message::text(String::from_utf8_lossy(&bytes));
+                            let message = Message::binary(bytes);
```

Comment:
I think we'll want to make this configurable. The change as-is is trading one assumption for another.

Could add a config option like `encode_as_binary : bool`

Or 🤔  , perhaps we could key off the configured encoder?

In your use case, what encoder are you using? For example, it seems like the `raw_message` codec would make sense to use binary frame with.

Reply:
I tend to agree. I think encoding `raw_message` as binary, but then `text` as UTF-8 makes sense to me.

Reply:
I use `raw_message` for my codec.

Reply:
@jszwedko @neuronull Can you take a look at the updated changes? I don't know anything about Rust, so please bear with my novice errors.

Reply:
> I tend to agree. I think encoding raw_message as binary, but then text as UTF-8 makes sense to me.

Were you thinking only apply the lossy function call if the encoder is `text` ?

Reply:
@jszwedko  - forgot to tag you on the above question. I realized there was some room for interpretation of your original comment. (Whether to encode everything that isn't `raw_message` , as UTF-8 lossy , or only `text`)

Reply:
👍 I was just drawing the distinction between `raw_message` and `text` encoders.  I believe`raw_message`, `avro`, and `native` should be binary, the rest should be UTF-8.

Reply:
Cool, thanks for clarifying.